screen/contributors/src/test/java/com/ivy/contributors/datasource/ContributorsDataSourceTest.kt

```diff
@@ -0,0 +1,121 @@
+package com.ivy.contributors.datasource
+
+import arrow.core.Either
+import com.ivy.base.TestDispatchersProvider
+import io.ktor.client.HttpClient
+import org.junit.Before
+import org.junit.Test
+import com.ivy.contributors.IvyWalletRepositoryDataSource.ContributorDto
+import com.ivy.contributors.ktor.IvyMockEngine
+import io.kotest.matchers.shouldBe
+import io.ktor.client.plugins.contentnegotiation.ContentNegotiation
+import io.ktor.serialization.kotlinx.json.json
+import io.mockk.coEvery
+import io.mockk.spyk
+import kotlinx.coroutines.test.runTest
+
+class ContributorsDataSourceTest {
```

Comment:
@rodrigoliveirac just use `mockk` for the unit tests

If you're creating integration tests there should be way simpler way. This seems like an overkill

Reply:
For reference:
https://github.com/Ivy-Apps/ivy-wallet/blob/main/shared/domain/src/androidTest/java/com/ivy/domain/usecase/SyncExchangeRatesUseCaseTest.kt

Reply:
> @rodrigoliveirac just use `mockk` for the unit tests
> 
> If you're creating integration tests there should be way simpler way. This seems like an overkill

Are you referring to the `FakeDataSource`? We can't use mockk with `HttpClient`. I'm updating the tests and to use the our real HttpClient.

Reply:
Why we can't mockk `HttpClient`? It's a bit harder but possibly. If you want to use the real HttpClient this must be an androidTest that runs on an emulator. Btw @rodrigoliveirac if it's too hard to add, it's fine leaving it w/o a test

Reply:
Because we're leading with this issue https://github.com/mockk/mockk/issues/571

Reply:
got it, thanks

Reply:
@rodrigoliveirac if you decide to make it an `androidTest` then you'll also have to the Integration Tests GitHub Actions workflow to run it